script.js

```diff
@@ -1,4 +1,72 @@
+var generateRandomInteger = function () {
+  var randomDec = Math.random() * 3;
+  var randomNum = Math.ceil(randomDec);
+
+  console.log(randomNum);
+  return randomNum;
+};
+
+var randomScissorPaperStone = function () {
+  var Num = generateRandomInteger();
+
+  if (Num == 1) {
+    return "scissors";
+  }
+
+  if (Num == 2) {
+    return "paper";
+  }
+
+  if (Num == 3) {
+    return "stone";
+  }
+};
+
+var generateResult = function (userPlays, computerPlays) {
+  var result = "Please enter scissors, paper or stone";
+
+  if (userPlays == computerPlays) {
+    result = "It's a tie!";
+  }
+
+  if (userPlays == "scissor" && computerPlays == "paper") {
+    result = "You win!";
+  }
+
+  if (userPlays == "scissors" && computerPlays == "stone") {
+    result = "You lose!";
+  }
+
+  if (userPlays == "paper" && computerPlays == "stone") {
+    result = "You win!";
+  }
+
+  if (userPlays == "paper" && computerPlays == "scissors") {
+    result = "You lose!";
+  }
+
+  if (userPlays == "stone" && computerPlays == "scissors") {
+    result = "You win!";
+  }
+
+  if (userPlays == "stone" && computerPlays == "paper") {
+    result = "You lose!";
+  }
```

Comment:
Consider using `||` so that you don't have to repeat the results, i.e. group all the win conditions into one `if` statement, and all the lose into one.

Reply:
Also consider using if/else if since scenarios are mutually exclusive.